src/constants/envVariables.tsx

```diff
@@ -1,4 +1,4 @@
 export const envVariables = {
   BASE_URL: import.meta.env.VITE_API_URL,
-  MAP_TOKEN: `https://api.mapbox.com/styles/v1/tobicorradi/cl3el2qne000o15pkcygzhmbq/tiles/{z}/{x}/{y}?access_token=${import.meta.env.VITE_MAP_ACCESS_TOKEN}`,
+  MAP_TOKEN: `https://api.mapbox.com/styles/v1/tobicorradi/cl3el2qne000o15pkcygzhmbq/draft/tiles/{z}/{x}/{y}?access_token=${import.meta.env.VITE_MAP_ACCESS_TOKEN}`,
```

Comment:
Why did you create this file?

I would have done only an environment file with the whole URL instead of being interpolating environment vars in a TSX file. And another thing, if you are rendering nothing in that file, it should be a TS file instead of a TSX. 

TSX files are used when you want to use JSX inside that file but you are not using it.